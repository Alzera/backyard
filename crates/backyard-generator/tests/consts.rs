use backyard_generator::generate;
use backyard_parser::parse_eval;

#[test]
fn basic() {
  let asts = parse_eval("const A = 1, B = 2;").unwrap();
  insta::assert_yaml_snapshot!(generate(&asts).unwrap());
}

#[test]
fn property() {
  let asts = parse_eval("class A { public const A = 1, B = 2; }").unwrap();
  insta::assert_yaml_snapshot!(generate(&asts).unwrap());
}
