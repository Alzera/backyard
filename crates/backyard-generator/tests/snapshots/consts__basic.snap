---
source: crates/backyard-generator/tests/consts.rs
expression: generate(asts).unwrap()
snapshot_kind: text
---
"const A = 1, B = 2;"
